src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -503,11 +518,133 @@ ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& m
                     "model and its config, please check passed config.");
 
     implement_properties();
+
     LOG_DEBUG("Done");
 }
 
-void ov::npuw::LLMCompiledModel::export_model(std::ostream& model) const {
-    OPENVINO_NOT_IMPLEMENTED;
+void ov::npuw::LLMCompiledModel::export_model(std::ostream& stream) const {
+    LOG_INFO("Serializing LLMCompiledModel...");
+    LOG_BLOCK();
+
+    using namespace ov::npuw::s11n;
+
+    // Serialize magic number first
+    write(stream, NPUW_SERIALIZATION_INDICATOR);
+
+    // Serialize general meta info
+    write(stream, OPENVINO_VERSION_MAJOR);
+    write(stream, OPENVINO_VERSION_MINOR);
+    write(stream, OPENVINO_VERSION_PATCH);
+
+    // Serialize name first + NPUW identifier for deserialization
+    // FIXME: consider a better approach then using name there
+    std::string name = std::string(NPUW_NAME_IDENTIFIER) + m_name;
+    write(stream, name);
```

Comment:
Why

Reply:
In constructor of CompiledModel and LLMCompiledModel I somehow need to identify that it's being called from deserialize(). So I need some mechanism to distinguish default constructor behavior from a dummy deserialized one

Reply:
Done